docs/book/book.toml

```diff
@@ -4,6 +4,7 @@ language = "en"
 multilingual = false
 src = "src"
 title = "The Sway Programming Language"
+google_analytics_id: "G-NEWVJ5V9T7"
```

Comment:
```suggestion
google_analytics_id= "G-NEWVJ5V9T7"
```

Reply:
Ahh, you're right!! thanks for the catch. Committing the suggestion.